src/shipit_code_coverage/default.nix

```diff
@@ -53,7 +53,7 @@ let
       hook = mkTaskclusterHook {
         name = "Shipit task aggregating code coverage data";
         owner = "mcastelluccio@mozilla.com";
-        schedule = [ "0 0 0 * * 0" ]; # every week
+        schedule = [ "0 0 * * * *" ]; # every day
```

Comment:
Shouldn't this be `0 0 0 * * *`? Isn't `0 0 * * * *` going to trigger the task every hour?

Reply:
You're right. The 6 field cron got me confused, because it's got that first field for seconds. Here's [a hourly schedule](https://github.com/mozilla/release-services/blob/87bd6580896314128abb07a040005cb23c61c21f/src/uplift/bot/default.nix#L23), this is not what we want.